LabExT/Movement/MoverNew.py

```diff
@@ -498,3 +570,23 @@ def _get_calibration(self, port=None, orientation=None, default=None):
             port)
         port = port or self._port_by_orientation.get(orientation)
         return self.calibrations.get((orientation, port), default)
+
+    def _wait_for_stages_to_stop(self, timeout: float = 2.0):
```

Comment:
Might be a very short timeout? Depending on the speed and distance, this time could be quite long?